xml/System.Net.Http/DelegatingHandler.xml

```diff
@@ -221,11 +221,20 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="request">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="request">The HTTP request message to send to the server.</param>
+        <param name="cancellationToken">A cancellation token to cancel operation.</param>
+        <summary>Sends an HTTP request to the inner handler to send to the server as a synchronous operation.</summary>
+        <returns>HTTP response message.</returns>
```

Comment:
Any chance you could expand the description of the return value? What does the HTTP response message represent in relation with the HTTP request that was sent?
```suggestion
        <returns>An HTTP response message.</returns>
```